package.json

```diff
@@ -29,7 +29,7 @@
     "conventional-changelog": "^1.1.0",
     "dateformat": "^1.0.11",
     "git-semver-tags": "^1.0.0",
-    "github": "^2.0.0",
+    "github": "^5.2.2",
```

Comment:
Thank you @alanzhou-okta!

I agree about keeping dependencies up-to-date. I'd love to see a follow up pull request to upgrade `github`.